src/renderers/shaders/ShaderChunk/tonemapping_pars_fragment.glsl.js

```diff
@@ -105,12 +113,39 @@ vec3 agxDefaultContrastApprox( vec3 x ) {
 
 }
 
-// AgX Tone Mapping implementation based on Filament, which in turn is based
-// on Blender's implementation using rec 2020 primaries
-// https://github.com/google/filament/pull/7236
-// Inputs and outputs are encoded as Linear-sRGB.
+vec3 agxLook( vec3 color, int look ) {
 
-vec3 AgXToneMapping( vec3 color ) {
+    if ( look == AGX_LOOK_BASE ) {
+
+        return color;
+
+    }
+
+    const vec3 lw = vec3( 0.2126, 0.7152, 0.0722 );
```

Comment:
I know this was copied from elsewhere, but these are not the correct luminance coefficients for BT.2020 primaries. I guess this can be addressed in a separate PR.

Reply:
I don't mind taking the time on this PR. I believe we're working in a log space at this point, between the inset and outset matrices. So the full pipeline would be:

Linear sRGB → Linear Rec.2020 → AgX Log → `<apply sigmoid and look>` → Linear Rec.2020 → Linear sRGB → sRGB

Reply:
Alternate version. 😅

![agx_pipeline](https://github.com/mrdoob/three.js/assets/1848368/eb8d3dcf-0499-4efa-9b3e-1f0a67c2fb71)

Reply:
Who knows, but I would like to think the color space remains Rec.2020.

Reply:
Hm... I hadn't yet noticed that these are BT.709 coefficients. Indeed, I don't understand why we'd be using BT.709 coefficients here, rather than BT.2020 coefficients (adapted to the log encoding or not).

These can be found in the [iodite "minimal agx" blog post](https://iolite-engine.com/blog_posts/minimal_agx_implementation), on which Godot's and Filament's implementations are also based. I'm not comfortable enough reading Blender's implementation, or Troy's original, to know how OpenColorIO implements this step.

Reply:
I believe these would be the luminance coefficients for Rec.2020.

    0.2626983  0.6780088  0.0592929

Common sense tells me these would be the appropriate coefficients.

Reply:
Before and after:

![bt2020_luma_coeffs](https://github.com/mrdoob/three.js/assets/1848368/3a00ddf2-e0f6-4bdc-8cd7-3e853b856e08)

I can't really tell the difference here, but I am confused, and I suspect there should be some accommodation for the log encoding. I've left a comment on the blog post asking about it.

Reply:
@WestLangley I received a response to [my question about the luminance coefficients](https://iolite-engine.com/blog_posts/minimal_agx_implementation) on the Minimal AgX blog past, and did some further reading. The coefficients come from [ASC CDL v1.2](https://en.wikipedia.org/wiki/ASC_CDL), which is a standardized representation of color grading looks. I haven't managed to find the actual text of ASC CDL v1.2, but the luminance coefficients are constant and based on Rec. 709 regardless of the working color space. This is not AgX-specific.


The benefits of matching the coefficients to the working color space here are unclear to me; the benefit of using Rec. 709 coefficients is the ability to import looks from other tools. I'm inclined to leave a comment about ASC-CDL v1.2 but to keep the coefficients of Rec. 709.